doc/hooks/memcached-test-hook.section.md

```diff
@@ -0,0 +1,50 @@
+
+# `memcachedTestHook` {#sec-memcachedTestHook}
+
+This hook starts a Memcached server during `checkPhase`. Example:
+
+```nix
+{ stdenv, memcachedTestHook }:
+stdenv.mkDerivation {
+
+  # ...
+
+  nativeCheckInputs = [
+    memcachedTestHook
+  ];
+}
+```
```

Comment:
Please format codeblocks with nixfmt-rfc-style.

Reply:
Fixed, as well as the redis hook docs